### DISCUSSION TITLE
Fluid Relay Service responding with 401 error when I call uploadBlob from my DataObject.

### DISCUSSION THREAD

**Discussion by lalavany-microsoft-com** [microsoft/FluidFramework#8756] [Q&A]
Hi,
[This is my sample project](https://github.com/lalavany-microsoft-com/collaborative-text-area-tutorial) where I am trying to share image data via fluid relay service.

When I call uploadBlob to FRS storage endpoint in MyDataObject, the request responds with 401 error. There isn't much help from the logs.

Can you please help me resolve this issue?

**Comment by tylerbutler** [comment]
401 is auth-related, so how does your request signing work? Are all the FRS requests failing, or just this one?

**Reply by lalavany-microsoft-com** [reply]
I am able to share other data like strings, only uploadBlob call is failing.
